living/hausa.md

```diff
@@ -4,29 +4,70 @@ Additional names:
 - 
 
 News:
-- 
-- 
+- [https://hausa.legit.ng/](https://hausa.legit.ng/)
```

Comment:
Please put URLs on a line on their own like this:
```md
- https://hausa.legit.ng/
```

See the Welsh.md file (linked in the readme) for an example.

Reply:
(This comment applies to multiple entries in this PR)

Reply:
fixed